### PR TITLE
Updating apiary examples

### PR DESCRIPTION
**Pull request by Tim-Jordison**
- added refresh-script that can be run to refresh all of the endpoint response bodies
- the pull request is rather immense looking. things worth reviewing are:
 - refresh-script.groovy
 - https://giftbitcurrencyapi113.docs.apiary.io/#reference (this is what the new responses will look like rendered in apiary)

### CODE REVIEW

File: apiary-sources/endpoint-refresh-script/refreshScript.groovy [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+Map calls = [:]
+for (Map call in json.calls) {
+    if (!call.callId) {
+        throw new Exception("No callId. Invalid input ${call}.")
+    }
+    if (call.get(call.callId)) {
+        throw new Exception("Invalid inout. Duplicate callId ${call.callId}")
+    }
+    if (!call.endpoint) {
+        throw new Exception("Invalid inpout. Call ${call}.")
```
Comment (tjukes) [inline apiary-sources/endpoint-refresh-script/refreshScript.groovy]: No pouting! Also error message above.

File: apiary-sources/endpoint-refresh-script/refreshScript.groovy [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+            }
+            if (replaceAsJson) {
+                value = JsonOutput.toJson(value)
+            }
+            println "replacement value: " + value
+            input = input.replace(replacement, value as String)
+        }
+        return input
+    } else {
+        println "this is happening"
```
Comment (tjukes) [inline apiary-sources/endpoint-refresh-script/refreshScript.groovy]: Could be a bit clearer here

File: apiary-sources/endpoint-refresh-script/README.md [hunk 3]
```diff
@@ -3,12 +3,11 @@ The point of this script is to be able to programatically update the response bo
...
 ## Steps to run the script
 1. Create a new account on lightraildev.net.
-2. Switch to LIVE mode and generate an API key.
-3. Paste the API key into the top of refreshScript.groovy.
-4. Run the script. If in intelliJ simply right click on it. From command line: `groovy refreshScript`
-5. Go to the ping output file and change the email to user@example.com. Specifically this file is `/Lightrail-API-Docs/endpoints/ping-GET.md`.
-6. Remove the API key since if it's committed it could be used to retrieve your email.
-7. Commit changes.  
+1. Switch to LIVE mode and generate an API key.
+1. Paste the API key into the top of refreshScript.groovy.
```
Comment (tjukes) [inline apiary-sources/endpoint-refresh-script/README.md]: This can probably go now?

Comment (Tim-Jordison) [reply apiary-sources/endpoint-refresh-script/README.md]: 👍

**Pull request merged**